crates/forge/tests/e2e/diagnostics_and_plugins.rs

```diff
@@ -1,44 +1,47 @@
 use crate::assert_stderr_contains;
-use indoc::indoc;
+use indoc::formatdoc;
 
 use crate::e2e::common::runner::{setup_package, test_runner};
 
 #[test]
 fn print_error_if_attributes_incorrect() {
     let temp = setup_package("diagnostics_and_plugins");
+    let temp_dir_path = temp.path().canonicalize().unwrap();
+    let temp_dir_path_str = temp_dir_path.to_str().unwrap();
```

Comment:
I'd name it `mock_tests_dir` or something like that, so it's obvious what it's testing.

`temp_dir` suggests we are still creating temp files when it's not the case anymore.